### PR TITLE
9444: Update EYB Sheet based off of 508 review

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to update the EYB bottom sheet implementation based on DEPO feedback so that it meets design and coding standards.

Note: AC 1, 2a, and 4 have already been handled by previous PRs

[Originating Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9444)

## Testing done
Testing passes locally

## Screenshots
N/A

## Acceptance criteria
- [x] 1. The estimated benefits drawer is a <button> + <div> that shows/hides a regular accordion.
- [x] 2. The following aria tags are added to the estimated benefits drawer:
  - [x] a. aria-expanded="false || true"
  - [x] b. aria-controls="UNIQUE_ID" The unique ID should be associated with the div of benefits.
  - [x] c. aria-controls should only appear when the drawer is open, otherwise it should be a null value (or we'll get an axe accessibility error)
- [x] 3. The estimated benefits drawer uses the prefers-reduced-motion browser functionality.
- [x] 4. Bottom sheet is displayed minimized on page load.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ export const EstimateYourBenefitsSummarySheet = props => (
   <div className="vads-u-padding-bottom--1p5 vads-u-border-top--1px vads-u-border-color--gray-light">
     <button
       aria-expanded={props.expandEybSheet ? 'true' : 'false'}
+      aria-controls={props.showEybSheet && 'eyb-sheet'}
```
Comment (zurbergram) [inline src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx:57]: will naming this `eyb-summary-sheet` cause an issues?

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 2]
```diff
@@ -390,6 +390,15 @@
     }
   }
 
+  @media (prefers-reduced-motion: reduce) {
```
Comment (zurbergram) [inline src/applications/gi/sass/partials/_gi-profile-page.scss:393]: can this be moved inside of `.eyb-sheet` ?

File: src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx (hunk 1 above)
Comment (elder-joseph) [reply src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx:57]: That should not cause any issues.

Comment (elder-joseph) [reply src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx:57]: That should not cause any issues

File: src/applications/gi/sass/partials/_gi-profile-page.scss (hunk 2 above)
Comment (elder-joseph) [reply src/applications/gi/sass/partials/_gi-profile-page.scss:393]: Will try moving that inside of `.eyb-sheet`

Comment (elder-joseph) [reply src/applications/gi/sass/partials/_gi-profile-page.scss]: That did not seem to cause issue either. Pushed a commit with both changes.

**Review by zurbergram — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**